### DISCUSSION TITLE
Romanian Translation

### DISCUSSION THREAD

**Discussion by LwveMike** [vuejs-translations/guidelines#81] [General]
So, I've seen that there are 2 more discussions about `Romanian translation` of the Vue documentation, but there were no updates for  2 years and it seems that the work is not started. I'm starting to translate the documentation and would appreciate any help with that or reviews on the translated pages. If you want to help leave a comment and I will create an Organisation. The forked repo is in my own repositories named `vuejs-docs-ro`

**Comment by Jinjiang** [comment]
@LwveMike it's very welcome to change a PR like this one #73 then people who have interest can come together to contribute. Thanks.
